cardshifter-core/src/main/java/com/cardshifter/core/groovy/EffectDelegate.groovy

```diff
@@ -14,6 +14,12 @@ import net.zomis.cardshifter.ecs.usage.functional.EntityConsumer
 
 import java.util.stream.Collectors
 
+/**
+ * Delegate for resolving effects.
+ *
+ * Each new effect should add a newline after adding their description. This helps separate effects from each other on
```

Comment:
This could be done by using a `String[]` instead of `StringBuilder`. Would probably be more elegant as we now split and join by newlines.